docs/introduction/core-concepts.md

```diff
@@ -57,7 +57,7 @@ const adapter = app => (event, dependencies) =>
 const handler = adapter => (event, context, callback) =>
```

Comment:
How about instead working with exports, just rename the handler to "laconia" here?

Reply:
The exports are the "actual" handler that AWS Lambda consumes, so I think it's better to name it handler here.